example/package.json

```diff
@@ -5,11 +5,12 @@
   "license": "MIT",
   "private": true,
   "dependencies": {
+    "@cratejoy/lightbox": "link:..",
     "prop-types": "^15.6.2",
     "react": "^16.4.1",
     "react-dom": "^16.4.1",
     "react-scripts": "^1.1.4",
-    "@cratejoy/lightbox": "link:.."
+    "styled-components": "^4.1.3"
```

Comment:
`peerDependencies`?

Reply:
Yeah, thats the example dir, they're peer'd on the real package that'll go to npm